crates/bevy_ecs/src/entity_disabling.rs

```diff
@@ -26,7 +26,17 @@ use crate::{
     component::{ComponentId, Components, StorageType},
     query::FilteredAccess,
 };
-use bevy_ecs_macros::Resource;
+use bevy_ecs_macros::{Component, Resource};
+
+#[cfg(feature = "bevy_reflect")]
+use {crate::reflect::ReflectComponent, bevy_reflect::Reflect};
+
+/// A marker component for disabled entities. See [the module docs] for more info.
+///
+/// [the module docs]: crate::entity_disabling
```

Comment:
Should we update the module docs to mention that the `Disabled` component is disabled in the default `DefaultQueryFilters`?  Right now they talk about it as a hypothetical.